portfolio/src/main/webapp/index.html

```diff
@@ -55,6 +55,22 @@ <h2>Past Projects</h2>
                 <li>Using Quicksort to sort data recursively</li>
                 <li>Reusing my Binary Search Tree to Create an AVL tree</li>
             </ul>
-            <p>Please refer to my GitHub if you would like to see the code behind my projects</p>
+            <p>Please refer to my <a href="github.com.lexidelorey" targer=_blank>
```

Comment:
1. I recommend prefixing the URL with `https://` so that it always opens the HTTPS version of the site. For github, it'll always be https, but just good practice.

2. Likely, your URL needs to be github.com/lexidelorey instead of github.com.lexidelorey.